openmmtools/mcmc.py

```diff
@@ -162,10 +172,55 @@ def apply(self, thermodynamic_state, sampler_state):
         sampler_state : openmmtools.states.SamplerState
            The initial sampler state before applying the move. This may
            be modified depending on the implementation.
+        context_cache : opemmtools.cache.ContextCache
```

Comment:
This is incomplete. It should be:
```
        context_cache_input : openmmtools.cache.ContextCache or None
            Context cache to be used in the propagation of the mcmc move. If None,
            it will create a new unlimited ContextCache object.
```